Add MP7K2 tournament article and update MP index page (#4327)
ppy/osu-wiki: wiki/Tournaments/MP/2020_7K/en.md

---
tags:

>   - MP7K
>   - MP7K2
>   - Druga Edycja
>   - Polish Championship

Reviewer: ```suggestion
  - MP7K
  - MP7K2
  - Polish Championship
```

---

# Mistrzostwa Polski w 7K 2020

> ![MP7K 2020 Logo](img/logo.png)

Reviewer: ```suggestion
![MP7K 2020 logo](img/logo.png)
```

> The **Mistrzostwa Polski w 7K 2020** (***MP7K 2020***, *7K Polish Championship 2020*) is a one-on-one, osu!mania 7-key tournament made for osu! players based in Poland or of Polish Heritage. It is the second installment of the tournament.

Reviewer: ```suggestion
The **Mistrzostwa Polski w 7K 2020** (***MP7K 2020***, *7K Polish Championship 2020*) is a 1v1 osu!mania 7-key tournament made for osu! players living in Poland or of Polish heritage. It is the second installment of the tournament.

```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |

> | Registration Phase | 2020-10-12/2020-10-25 |
> | Qualifiers Showcase | 2020-10-31 |

Reviewer: ```suggestion
| Registration phase | 2020-10-12/2020-10-25 |
| Qualifiers showcase | 2020-10-31 |

```

| Qualifiers | 2020-11-07/2020-11-08 |
| Group Stage | 2020-11-14/2020-11-15 |
| Round of 16 | 2020-11-21/2020-11-22 |
| Quarterfinals | 2020-11-28/2020-11-29 |
| Semifinals | 2020-12-05/2020-12-06 |
| Finals | 2020-12-12/2020-12-13 |
| Grand Finals | 2020-12-19/2020-12-20 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | 150 PLN |
> | ![Silver Crown](/wiki/shared/crown-silver.png "2nd place") | 100 PLN |
> | ![Bronze Crown](/wiki/shared/crown-bronze.png "3rd place") | 50 PLN |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 150 PLN |
| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 100 PLN |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 50 PLN |

```

## Organisation

| Position | Member(s) |
| :-- | :-- |
| Host | ![][flag_PL] [Benzopirene](https://osu.ppy.sh/users/1887068), ![][flag_PL] [Otis1204](https://osu.ppy.sh/users/11431349) |

> | Map Selectors | ![][flag_PL] [Benzopirene](https://osu.ppy.sh/users/1887068), ![][flag_PL] [DaDarkDragon](https://osu.ppy.sh/users/8902097), ![][flag_PL] [Otis1204](https://osu.ppy.sh/users/11431349), ![][flag_PL] [paprotkaaaa](https://osu.ppy.sh/users/7233032) |

Reviewer: ```suggestion
| Map selector | ![][flag_PL] [Benzopirene](https://osu.ppy.sh/users/1887068), ![][flag_PL] [DaDarkDragon](https://osu.ppy.sh/users/8902097), ![][flag_PL] [Otis1204](https://osu.ppy.sh/users/11431349), ![][flag_PL] [paprotkaaaa](https://osu.ppy.sh/users/7233032) |

```

> | Referees | ![][flag_PL] [Benzopirene](https://osu.ppy.sh/users/1887068), ![][flag_PL] [capsafangirl](https://osu.ppy.sh/users/4308015), ![][flag_PL] [crafciak31](https://osu.ppy.sh/users/8426128), ![][flag_PL] [DaDarkDragon](https://osu.ppy.sh/users/8902097), ![][flag_PL] [Eclipse-](https://osu.ppy.sh/users/8493070), ![][flag_PL] [Justin14p](https://osu.ppy.sh/users/8461297), ![][flag_PL] [SitekX](https://osu.ppy.sh/users/3840946), ![][flag_PL] [szuncoll](https://osu.ppy.sh/users/12393014), ![][flag_PL] [Yamakazi](https://osu.ppy.sh/users/4726691) |

Reviewer: ```suggestion
| Referee | ![][flag_PL] [Benzopirene](https://osu.ppy.sh/users/1887068), ![][flag_PL] [capsafangirl](https://osu.ppy.sh/users/4308015), ![][flag_PL] [crafciak31](https://osu.ppy.sh/users/8426128), ![][flag_PL] [DaDarkDragon](https://osu.ppy.sh/users/8902097), ![][flag_PL] [Eclipse-](https://osu.ppy.sh/users/8493070), ![][flag_PL] [Justin14p](https://osu.ppy.sh/users/8461297), ![][flag_PL] [SitekX](https://osu.ppy.sh/users/3840946), ![][flag_PL] [szuncoll](https://osu.ppy.sh/users/12393014), ![][flag_PL] [Yamakazi](https://osu.ppy.sh/users/4726691) |

```

> | Streamers |  ![][flag_PL] [Dve](https://osu.ppy.sh/users/1807700) |

Reviewer: ```suggestion
| Streamer |  ![][flag_PL] [Dve](https://osu.ppy.sh/users/1807700) |

```

> | Commentators | ![][flag_PL] [Archaic84](https://osu.ppy.sh/users/8611177), ![][flag_PL] [BarTkE](https://osu.ppy.sh/users/5004339), ![][flag_PL] [capsafangirl](https://osu.ppy.sh/users/4308015), ![][flag_PL] [DaDarkDragon](https://osu.ppy.sh/users/8902097), ![][flag_PL] [grill](https://osu.ppy.sh/users/12352929), ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783), ![][flag_PL] [Otis1204](https://osu.ppy.sh/users/11431349), ![][flag_PL] [paprotkaaaa](https://osu.ppy.sh/users/7233032), ![][flag_PL] [SitekX](https://osu.ppy.sh/users/3840946), ![][flag_PL] [TheFalek](https://osu.ppy.sh/users/10214260), ![][flag_PL] [Tidek](https://osu.ppy.sh/users/743282) |

Reviewer: ```suggestion
| Commentator | ![][flag_PL] [Archaic84](https://osu.ppy.sh/users/8611177), ![][flag_PL] [BarTkE](https://osu.ppy.sh/users/5004339), ![][flag_PL] [capsafangirl](https://osu.ppy.sh/users/4308015), ![][flag_PL] [DaDarkDragon](https://osu.ppy.sh/users/8902097), ![][flag_PL] [grill](https://osu.ppy.sh/users/12352929), ![][flag_PL] [Kamikaze](https://osu.ppy.sh/users/2124783), ![][flag_PL] [Otis1204](https://osu.ppy.sh/users/11431349), ![][flag_PL] [paprotkaaaa](https://osu.ppy.sh/users/7233032), ![][flag_PL] [SitekX](https://osu.ppy.sh/users/3840946), ![][flag_PL] [TheFalek](https://osu.ppy.sh/users/10214260), ![][flag_PL] [Tidek](https://osu.ppy.sh/users/743282) |

```

> | Graphic Designer | ![][flag_PL] [capsafangirl](https://osu.ppy.sh/users/4308015), ![][flag_PL] [W D Gaster](https://osu.ppy.sh/users/1475814) |

Reviewer: ```suggestion
| Graphic designer | ![][flag_PL] [capsafangirl](https://osu.ppy.sh/users/4308015), ![][flag_PL] [W D Gaster](https://osu.ppy.sh/users/1475814) |

```

## Links

- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1160806)
- [Livestream](https://www.twitch.tv/polishosumania)
- [MP Discord server](https://discord.gg/0yb1K4q04lERYyUv)

---

## Mappools

### Qualifiers

> Mappool for Qualifiers is going to be published 2020-10-31.

Reviewer: ```suggestion
*to be published on 2020-10-31*
```
might be fine?
